src/main/java/com/wishlist/model/Product.java

```diff
@@ -0,0 +1,25 @@
+package com.wishlist.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "product")
```

Comment:
Some SQL dialects require in SQLs to use quote marks when table name has capital letters